index.js

```diff
@@ -0,0 +1,53 @@
+const arr = JSON.parse(localStorage.getItem('formData')) || [];
+
+const form = document.querySelector('#form');
+const add = document.getElementById('addBtn');
+// const title = document.getElementById('title');
+// const author = document.getElementById('author');
```

Comment:
- [OPTIONAL] Following the best practices, it is not advisable to leave unused code as a comment in your codebase. It makes your codebase look unprofessional. :smile: :+1: I'll suggest you remove this.